src/lib/eliom_notif.server.ml

```diff
@@ -188,3 +217,29 @@ module Make (A : S) = struct
     Lwt.return @@ Notif_hashtbl.iter f I.tbl
 
 end
+
+module type SIMPLE = sig
+  type identity
+  type key
+  type notification
+  val equal_key                  : key -> key -> bool
+  val equal_identity             : identity -> identity -> bool
+  val get_identity               : unit -> identity Lwt.t
+  val max_resource               : int
+  val max_identity_per_resource  : int
```

Comment:
Can we simplify this a bit more with sane defaults for `max_*` and maybe even `(=)` for `equal_*`? There is always `Make` for the more involved cases.